test/integration/apiserver/apply/apply_test.go

```diff
@@ -1416,3 +1418,198 @@ func TestClearManagedFieldsWithUpdate(t *testing.T) {
 		t.Fatalf("Expected other fields to stay untouched, got: %v", object)
 	}
 }
+
+func TestApplyStatusPod(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.ServerSideApply, true)()
+
+	_, client, closeFn := setup(t)
+	defer closeFn()
+
+	_, err := client.CoreV1().RESTClient().Patch(types.ApplyPatchType).
+		Namespace("default").
+		Resource("pods").
+		Name("test-pod").
+		Param("fieldManager", "apply_test").
+		Body([]byte(`{
+			"apiVersion": "v1",
+			"kind": "Pod",
+			"metadata": {
+				"name": "test-pod"
+			},
+			"spec": {
+				"containers": [{
+					"name":  "test-container",
+					"image": "test-image"
+				}]
+			}
+		}`)).
+		Do(context.TODO()).
+		Get()
+	if err != nil {
+		t.Fatalf("Failed to create object using Apply patch: %v", err)
+	}
+
+	_, err = client.CoreV1().RESTClient().Patch(types.ApplyPatchType).
```

Comment:
is there a reason not to use the typed client?
```go
client.CoreV1().Pods("default").Patch(
  context.TODO(),
  "test-pod",
  types.ApplyPatchType,
  []byte(`...`),
  metav1.PatchOptions{FieldManager: "apply_test"},
  "status",
)
```

Reply:
Paradox of choice :-)